codice/ddf
# DDF-04303 Updated "Catalog UI Search" to "Catalog Search UI"

#### What does this PR do?

- Removes some leftover references to the Standard Search UI
- Renames the catalog-ui property to catalog-search-ui
- Reorders application name property replacements to be defined together
- Updates "Catalog UI Search" to "Catalog Search UI"
- Renames Search UI features to be consistent

#### Who is reviewing it? 
@Lambeaux @brjeter @bdeining @stustison 
#### Select relevant component teams: 
@codice/docs 
#### Ask 2 committers to review/merge the PR and tag them here.
@ricklarsen - Documentation
@vinamartin
@pklinef
#### How should this be tested?

- Confirm that you can use both the Simple Search UI and Standard Search UI after a default install.
- Confirm the configurations in `Admin Console` -> `Search UI` -> `Configuration` say `Catalog Search UI` instead of `Catalog UI Search`.
- Search the documentation ([attached](https://github.com/codice/ddf/files/2927562/documentation.html.zip)) for any references to a Search UI and make sure that they are consistent. Confirm that that the names of the configurations for Intrigue match between the docs and Admin Console.
- Confirm that downstream projects weren't broken.

#### Any background context you want to provide?
There will be another PR to add the `Catalog UI Search` prefix to other configuration names under `Admin Console` -> `Search UI` -> `Configuration`.
#### What are the relevant tickets?
https://github.com/codice/ddf/issues/4303

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in-line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
